substrate/frame/staking/src/lib.rs

```diff
@@ -334,6 +334,10 @@ pub use pallet::{pallet::*, UseNominatorsAndValidatorsMap, UseValidatorsMap};
 pub(crate) const STAKING_ID: LockIdentifier = *b"staking ";
 pub(crate) const LOG_TARGET: &str = "runtime::staking";
 
+// Marker type to pass as type [`sp_staking::OnStakingUpdate`] that does nothing, i.e. it does not
+// propagate the event to any event listener.
+pub(crate) type UntrackedEvent = ();
```

Comment:
not sure if this branch is not in a consistent state, this type is still around, but there is no logic around it? We are simply dropping events?

Reply:
We still want to postpone the event to be emitted when the nominators are rewarded until we enable the paged rewards. I will rename this to not be confusing.

Reply:
I removed all the code and comments related to the stake tracker stuff.